README.md

```diff
@@ -98,6 +98,11 @@ Then, elsewhere in the library:
 use crate::sync::AtomicUsize;
 ```
 
+If you have tokio dependencies in your project, you may wish to choose a cfg flag 
```

Comment:
May be worth saying 
```suggestion
If you have any dependencies in your project which also use `loom`, you may wish to choose a cfg flag 
```
or similar? It's definitely possible for similar issues to occur with other libraries...

Reply:
Good point!